runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -319,6 +319,7 @@ void setupSwaps()
 		swap("wear", "rub", config::swapTeleportItem);
 		swap("wear", "teleport", config::swapTeleportItem);
 		swap("wield", "teleport", config::swapTeleportItem);
+		swap( "wield", "invoke", config::swapTeleportItem);
```

Comment:
I'll commit this via GitHub's UI
```suggestion
		swap("wield", "invoke", config::swapTeleportItem);
```